cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterDescriptor.java

```diff
@@ -204,4 +206,53 @@ private void loadProps() {
     }
     return urlList;
   }
+
+  public void loadHotModifiedProps() throws QueryProcessException {
+    String url = getPropsUrl();
+    if (url == null) {
+      return;
+    }
+    try (InputStream inputStream = new FileInputStream(new File(url))) {
+      logger.info("Start to reload config file {}", url);
+      Properties properties = new Properties();
+      properties.load(inputStream);
+      loadHotModifiedProps(properties, false);
+    } catch (Exception e) {
+      logger.warn("Fail to reload config file {}", url, e);
+      throw new QueryProcessException(
+          String.format("Fail to reload config file %s because %s", url, e.getMessage()));
+    }
+  }
+
+  public void loadHotModifiedProps(Properties properties, boolean toCheckProperties)
+      throws QueryProcessException {
+    if (toCheckProperties && !checkProperties(properties)) {
+      throw new QueryProcessException(
+          "Failed to load configuration from properties because some are missing locally.");
+    }
+    // TODO implement some hot modified properties to load
+    logger.info("Set cluster configuration {}", properties);
```

Comment:
Please find somewhere to add the documents about what configurations are supported even if it is only an empty list now.